paperoni/sql/collection.py

```diff
@@ -0,0 +1,608 @@
+"""Collection class to manage a SQL database."""
+import pprint
+from typing import List, Union, Optional
+from paperoni.papers2 import Paper, Link, Author, Venue, Release, Topic
+from paperoni.sql.database import Database
+from paperoni.utils import get_venue_name_and_volume
+
+
+class MutuallyExclusiveError(RuntimeError):
+    """
+    Exception raised when concurrent parameters are passed to
+    Collection.query().
+    """
```

Comment:
This exception should be caught at the top level so that a proper informative error message can be displayed. Currently it just prints a traceback.